controllers/node.go

```diff
@@ -758,3 +764,25 @@ func runForceServerUpdate(node *models.Node) {
 func isServer(node *models.Node) bool {
 	return node.IsServer == "yes"
 }
+
+func updateRelay(oldnode, newnode *models.Node) {
+	relay := logic.FindRelay(oldnode)
+	newrelay := relay
+	if oldnode.Address != newnode.Address {
```

Comment:
I'd recommend putting a comment `if oldnode.Address != newnode.Address { // check for address difference to...` for future reference